src/Symfony/Component/Messenger/Bridge/Amqp/Transport/AmqpStamp.php

```diff
@@ -45,7 +46,12 @@ public function getAttributes(): array
         return $this->attributes;
     }
 
-    public static function createFromAmqpEnvelope(\AMQPEnvelope $amqpEnvelope, self $previousStamp = null): self
+    public function isRetryAttempt(): bool
```

Comment:
Lets add this new method below the named factory method. (createFromAmqpEnvelope)